Cargo.toml

```diff
@@ -30,7 +30,7 @@ ff = "0.13"
 fpe = "0.5"
 group = { version = "0.13", features = ["wnaf-memuse"] }
 halo2_gadgets = "0.2"
-halo2_proofs = "0.2"
+halo2_proofs = { version = "0.2", default-features = false, features = ["batch"] }
```

Comment:
This makes the `halo2-batch` feature flag below unusable, because it marks `halo2_proofs/batch` as a required feature flag. But if WASM is working with this PR, that means the `halo2-batch` feature flag is unnecessary.